ci/travis-ci.sh

```diff
@@ -23,7 +23,12 @@ export MAYACTL="$GOPATH/src/github.com/openebs/maya/bin/maya/mayactl"
 ./ci/build-maya.sh
 rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
 
-curl https://raw.githubusercontent.com/openebs/openebs/master/k8s/ci/test-script.sh > test-script.sh
+#curl https://raw.githubusercontent.com/openebs/openebs/master/k8s/ci/test-script.sh > test-script.sh
+curl https://raw.githubusercontent.com/openebs/openebs/fb0100f6a2f343b3c77e52f922dd40f0e7ded417/k8s/ci/test-script.sh > test-script.sh
```

Comment:
why this change? Why we are picking the file from a particular commit?

Reply:
Just to make sure that Travis should pass with the test case added. It will be reverted back once got approvals.